USFMToolsSharp.Renderers.HTML.Tests/HTMLRenderTest.cs

```diff
@@ -102,7 +102,7 @@ public void TestTableRender()
             Assert.AreEqual("<div><table class=\"table-block\"><tr><td class=\"table-cell\">dari suku Ruben</td><td class=\"table-cell-right\">12.000</td></tr></table></div>", WrapTest("\\tr \\tc1 dari suku Ruben \\tcr2 12.000"));
 
             // Embedded Verse
-            Assert.AreEqual("<td class=\"table-cell\"><span class=\"verse\"><sup class=\"versemarker\">6</sup>dari suku Asyer</span></td>", WrapTest("\\tc1 \\v 6 dari suku Asyer"));
+            Assert.AreEqual("<td class=\"table-cell\"></td><span class=\"verse\"><sup class=\"versemarker\">6</sup>dari suku Asyer</span>", WrapTest("\\tc1 \\v 6 dari suku Asyer"));

```

Comment:
This seems like a mistake -- the verse is outside the `<td>...</td>` marker.

Is this intentional?

Reply:
It was changed in the main parser, it was a quite a while ago but there must have been a reason.